bitrepository-webclient/src/main/webapp/integrity-service.html

```diff
@@ -267,23 +269,34 @@ <h3 id="modalPagerLabel">Modal header</h3>
             context.title = type + " on " + id;
             context.member = "checksumErrorCount";
             context.url += "getChecksumErrorFileIDs/";
+        } else if(type == "Checksum age limit") {
+          context.element = id + "-maxAgeForChecksums";
+          context.title = type + " on " + id;
+          context.member = "maxAgeForChecksums";
+        } else if(type == "Oldest checksum age") {
+          context.element = id + "-ageOfOldestChecksum";
+          context.title = type + " on " + id;
+          context.member = "ageOfOldestChecksum";
```

Comment:
Rest of the JavaScript is 4-space indented so just stick to that. I know the above HTML is 2-space indented and it is weird to have two different indentation styles in one file, but this will be fixed when we separate the two into their own files.

Reply:
I was too lazy and let my IntelliJ IDEA do the indentation. Will fix.